src/librustc_trans/trans/cabi_powerpc64le.rs

```diff
@@ -0,0 +1,245 @@
+// Copyright 2014-2016 The Rust Project Developers. See the COPYRIGHT
```

Comment:
There are differences between the BE and LE ABI when it comes to returning aggregates. As an example:

```
struct foo {
	float a[8];
};

struct foo blah(struct foo x)
{
	return x;
}
```

On big endian we pass the return value via a pointer in r3:

```
blah:
	std 4, 0(3)
	std 5, 8(3)
	std 6, 16(3)
	std 7, 24(3)
	blr
```

On little endian we pass it via registers:

```
blah:
	blr
```